src/Symfony/Component/Validator/Tests/Constraints/UrlValidatorTest.php

```diff
@@ -67,11 +67,14 @@ public function getValidUrls()
     {
         return [
             ['http://a.pl'],
+<<<<<<< HEAD
```

Comment:
Looks like a rebase that went wrong. Can you fix it?

Reply:
conflicts resolved